caffe2/operators/utility_ops.cc

```diff
@@ -726,17 +726,6 @@ REGISTER_GRADIENT(Alias, GetAliasGradient);
 
 SHOULD_NOT_DO_GRADIENT(ResizeLike);
 
-class GetSumGradient : public GradientMakerBase {
```

Comment:
hmm, we also have an old SumGradient here... We need to make the new one backward compatible with the old one. To achieve this, you probably need to check the input number and output number in the gradient op, and decide which version of the operator we are handling...